### PR TITLE
moved the installation guides into a special folder and renamed them.…

### PR DESCRIPTION
**Pull request by mrmeier-1**
… Still need to check the links 😅. Also added a page at the end, called What's Next

### CODE REVIEW

File: docs/versioned_docs/version-0.13.1/general/02-Components/AskUI-Development-Environment.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 3
```
[the remark this answers is not in the document]
Comment (programminx-askui) [reply docs/versioned_docs/version-0.13.1/general/02-Components/AskUI-Development-Environment.md:2]: I don't want to correct twice.

**Review by programminx-askui — CHANGES_REQUESTED** [review]
Hello, 

I've reviewed the the docs and my main findings are:
- Renaming issues
  -  `Askui-*` to `AskUI-*`
  - Renamings from parameters
- Unused Images from the Installer

My question is, how are we proceeding from here?

File: docs/versioned_docs/version-0.13.1/general/02-Components/AskUI-Development-Environment.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JohannesDienst-askui) [reply docs/versioned_docs/version-0.13.1/general/02-Components/AskUI-Development-Environment.md:2]: We have to copy the contents to 0.13.1 so it is the default when users get to the docs.

File: docs/sidebars.js [hunk 2]
```diff
@@ -14,19 +14,137 @@
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
-  docsSidebar: [{ type: 'autogenerated', dirName: 'general' }],
+  // docsSidebar: [{ type: 'autogenerated', dirName: 'general' }],
```
[the remark this answers is not in the document]
Comment (JohannesDienst-askui) [reply docs/sidebars.js:17]: No, this is intended as we do need to define the Sidebar manually.

File: docs/blog/2024-02-07/2024-02-07.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+3. **Configuration (First-Time Usage):**
+    - Configure AskUI settings:
+        1. Type **`AskUI-SetSettings -Token <askui_token> -WorkspaceId <askui_workspace_id>`**.
+        2. (Optional for Proxy Users) Type **`Askui-SetSettings -HttpProxy <proxy_http_address> -HttpsProxy <proxy_https_address>`**.
+4. **Verify Settings:**
+    - Type **`AskUI-ShowSettings`** to verify the configured settings.
+5. **Start AskUI Controller:**
+    - Start the AskUI Controller in background mode by typing **`AskUI-StartController -RunInBackground`**.
+6. **Create a New Project:**
+    - Create a new project by typing **`AskUI-NewProject`**.
```
[the remark this answers is not in the document]
Comment (JohannesDienst-askui) [reply docs/blog/2024-02-07/2024-02-07.md:113]: Added a note that explains this.

**Review by programminx-askui — APPROVED** [review]

**Pull request merged**